app/src/main/res/values/strings.xml

```diff
@@ -1552,6 +1573,18 @@ In group conversations, the group admin can overwrite this setting.</string>
         <item quantity="one">1 second</item>
         <item quantity="other">%1$d seconds</item>
     </plurals>
+    <plurals name="days_label">
+        <item quantity="one">day</item>
+        <item quantity="other">days</item>
+    </plurals>
+    <plurals name="weeks_label">
+        <item quantity="one">week</item>
+        <item quantity="other">weeks</item>
+    </plurals>https://nginz-https.bund-next-column-1.wire.link/deeplink.html
```

Comment:
```suggestion
    </plurals>
```

Reply:
Whoops, nice catch, it got copied in a wrong place 😅